app/screens/GameScreen.tsx

```diff
@@ -0,0 +1,126 @@
+import { useNavigation } from "expo-router";
+import React, { useState } from "react";
+import {
+  Dimensions,
+  Image,
+  Pressable,
+  StyleSheet,
+  Text,
+  TouchableOpacity,
+  View,
+} from "react-native";
+
+//pentru a optimiza si usura urmarirea codului , ar fi necesara folosirea state ului pentru lives
```

Comment:
EN and patterns abobe

For larger comments you should use the 

/**
* Comment
*/

This will also appear when you hover the component